examples/example_matplotlib.ipynb

```diff
@@ -0,0 +1,214 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0f4c7d83-4f5e-48f7-abe7-57997f6dc4a6",
+   "metadata": {},
+   "source": [
+    "# Example - Matplotlib Integration"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2551617f-7126-4b7c-ad0b-4ec5c38a39fe",
+   "metadata": {},
+   "source": [
+    "## Import libraries and setup node_editor"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "938e9808-8acd-4598-a3bd-9bea0bb39d48",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "import jupyterlab_nodeeditor as jlne\n",
+    "from yggdrasil import yamlfile\n",
+    "import matplotlib.pyplot as plt\n",
+    "import ipywidgets as widgets\n",
+    "import numpy as np"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7ea24a11-1036-4cba-80ca-75c3a95fa572",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Setting up the JLNE instance\n",
+    "filename = \"X:\\College\\Grad School\\Research\\gc-Xyzic\\jupyterlab_nodeeditor\\examples\\model_example.yml\"\n",
```

Comment:
This is not necessary for this pull request, but this is not as portable as using relative pathnames.  We can infer the pathname from the filename we're currently in.

Reply:
I think this notebook needs an update as I don't remember what that filepath even led to and I think it uses the outdated parser? Let me know if you want me to work on this.